Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -284,6 +284,8 @@ Two-factor authentication (2FA) is standard for apps that allow users to access
 - Hardware or software token
 - Push notifications in combination with PKI and local authentication
 
+Whatever option is used as 2nd factor, it always must be enforced and verified on the server-side and never on client-side. Otherwise the 2nd factor can easily be bypassed within the app.
```

Comment:
```suggestion
Whatever option is used as 2nd factor, it always must be enforced and verified on the server-side and never on client-side. Otherwise the 2nd factor can be easily bypassed within the app.
```